packages/indexer/Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM rust:1.76-bookworm
-
-RUN apt-get update && apt-get install -y cmake clang
-
+FROM rust:1.80-alpine3.19 as build 
+RUN apk add --no-cache git cmake clang libressl-dev
 WORKDIR /
 RUN git clone --depth 1 --branch v1.0.0-beta.4  https://github.com/dashevo/platform
-
 WORKDIR /app
 COPY Cargo.lock /app
 COPY Cargo.toml /app
 COPY migrations /app/migrations
 COPY src /app/src
-
-RUN cargo build
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --target x86_64-unknown-linux-musl --release
 RUN cargo install refinery_cli
+
+FROM scratch as indexer
```

Comment:
Why scratch? I see we use alpine in the build section, maybe do here it too? I'm afraid scratch can have potential issues if indexer would use some feature from a static library which is not provided in the scratch by default. I think it's better to just use alpine

Reply:
✅